source/guides/ember-cli/index.md

```diff
@@ -0,0 +1,56 @@
+Ember comes with a toolchain called Ember CLI that provides everything you need while developing your app behind a single command:  ```ember```.
+
+* State of the art asset management (use ```ember build``` to build your entire project)
+* Quick scaffolding like ```ember generate route some-page```
```

Comment:
This sentence read awkwardly to me. I'm afraid that people who don't know what "scaffolding" is won't be aided by the `ember generate…` example. Maybe we can briefly describe the benefits? Something like:

> Built-in generators streamline your workflow by creating ready-made files for components, models, routes and more.